shared/js/manifest_helper.js

```diff
@@ -51,6 +51,10 @@ function ManifestHelper(manifest) {
   }
 }
 
+ManifestHelper.prototype.getShortNameOrName = function() {
```

Comment:
I personally would like to see a more generic function name here, so consumers of this don't have to update in case we change this in the future.

My preference would be to change this to something like ```displayName``` or ```caption```. Thoughts?